content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -634,83 +636,52 @@ However, we don't have an vertex shader input that gives us the previous positio
-6. Ensure you are viewing your tree. Note that you probably won't see much except black on the bottom left preview, because the motion vectors are very small because our tree only moves minimally. However, if you move the camera around or translate the tree quickly, you may see some motion vectors pop up. 
-   
-   You can also open `MeshMotionVectorVegetationBending.azsli` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
+1. Press the **Home** key on your keyboard. This brings up the toolbar at the top.
+1. Select **Atom Tools** > **Pass Viewer**.
+1. In the pop-up **PassTree View**, enable **Preview Attachment** and **Show Pass Attachments**.
+1. In the **PassTree**, find *MotionVectorPass* > *MeshMotionVectorPass* and select the line with `CameraMotion`. 
+1. Ensure you are viewing your tree. Note that you probably won't see much except black on the bottom left preview, because the motion vectors are very small because our tree only moves minimally. However, if you move the camera around or translate the tree quickly, you may see some motion vectors pop up. You can also open `MeshMotionVectorVegetationBending.azsl` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
 
 This video shows the motion vectors when `OUT.m_motion` is scaled by `10000.0`.
 {{< video src="/images/atom-guide/vegetation-bending-tutorial/motionvectortree.mp4" autoplay="true" loop="true" muted="true" width="100%" info="Video of tree motion vector." >}}
 
 The pass tree visualizer tool is also helpful with debugging shaders and passes, so you can see the output of certain steps of different passes when you select them in the **PassTree**.
 
+## Download the AtomTutorial Gem sample
+Now that you've completed this tutorial, you can compare your results to our working version of the vegetation bending material type in the **AtomTutorials** gem in the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). You can either download and place the [final working vegetation bending files]() in your project, or you can download the gem and add it to the engine.
+
+If you'd like to download and enable the **AtomTutorials** gem, do the following:
+1. Download or clone the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). 
+1. Open `VegetationBending.materialtype` and replace all the instances of `{your-path-to-o3de}` with your absolute path to O3DE. 
+1. [Register the **AtomTutorials** gem to the engine](/docs/user-guide/project-config/register-gems). In the command line interface, `cd` into the engine. Then, run
+   ```
+   scripts\o3de register -gp {your-path-to-sample-code-gems}\atom_gems\AtomTutorials -espp {your-project-path}
+   ```
+   For example, `scripts\o3de register -gp C:\sample-code-gems\atom_gems\AtomTutorials -espp C:\MyProject`
```

Comment:
I think you can do this step through o3de.exe too, I remember seeing an Add Gem button.

Reply:
Hmm, I can find adding the gem but I can't seem to find an option to register a new gem. The docs (https://www.o3de.org/docs/user-guide/project-config/register-gems/) also don't mention using o3de.exe, so I thought it'd be safer to just provide the CLI commands.